tests/foreman/api/test_smartproxy.py

```diff
@@ -48,13 +47,13 @@ def module_proxy_attrs(module_target_sat):
     return set(smart_proxy.update_json([]).keys())
 
 
-def _create_smart_proxy(request, **kwargs):
+def _create_smart_proxy(request, target_sat, **kwargs):
     """Create a Smart Proxy and add the finalizer"""
     proxy = entities.SmartProxy(**kwargs).create()
```

Comment:
```suggestion
    proxy = target_sat.api.SmartProxy(**kwargs).create()
```

Reply:
So does broadly preferring to use target_sat over entities make sense

Reply:
Yep, great to update when and where we can, but its not strictly required yet.